README.md

```diff
@@ -1,174 +1,166 @@
+| profile | string | Use a specific profile from [AWS CLI credential file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) |
+| range | string | Specify range header for GET requests |
+| ratelimit | float | The total number of operations per second across all threads. Default: `1.7976931348623157e+308` | 
+| region | string | Region to send requests to. Default: `us-east-1` |
+| repeat | int | Repeat each S3 operation this many times: Default: `0` (do not repeat) |
+| requests | int | Total number of requests. Mutually exclusive with `duration`. Default: `1000` |
+| retrysleep | int | How long to sleep in between each retry in milliseconds. Default: `0` (exponential backoff) |
+| rr | - | Reduced redundancy storage for PUT requests | 
+| size | int | Object size in bytes. Note that s3tester is not ideal for very large objects as the entire body must be read for v4 signing and the AWS SDK does not support v4 chunked. Performance may degrade as size increases due to the use of v4 signing without chunked support. Default: `30720` |
+| tagging | string | The tag-set for the object. The tag-set must be formatted as such: `'tag1=value1&tage2=value2'`. Used for `put`, `puttagging`, `putget` and `putget9010r` |
+| tier | string | The retrieval option for restoring an object. One of `expedited`, `standard`, or `bulk`. AWS default option is standard if not specified. Default: `standard` |
+| uniformDist | string | Generates a uniform distribution of object sizes given a min-max size. Allowed values: `10` to `20`) |
+| verify | int | Verify the retrieved data on a get operation. `0`: disable verify (default); `1`: normal put data, `2`: multipart put data. If verify equals `2`, partsize is required (default `partsize` is `5242880` bytes) |
+| workload | string | Filepath to JSON either a replay file generated by the auditAnalysis tool to play an exact workload on a grid or a `mixedWorkload` JSON file which allows a user to specify a mixture of operations. A sample mixed workload file must be in the format provided below the table |
+
+#### mixedWorkload JSON Sample File
+
+```raw
+{
+	"mixedWorkload": [{
+		"operation": "put",
+		"ratio": 25
+	}, {
+		"operationType": "get",
+		"ratio": 25
+	}, {
+		"operationType": "updatemeta",
+		"ratio": 25
+	}, {
+		"operationType": "delete",
+		"ratio": 25
+	}]
+}
+```
+
+**NOTE:** The order of operations specified will generate the requests in the same order. That is, if you have DELETE followed by a PUT, but no objects on your grid to delete, all your deletes will fail.
+
+## Exit codes
+
+- `1`: one or more requests has failed
+
+## Examples
+
+### Writing objects into a bucket
+
+```raw
+./s3tester -concurrency=128 -size=20000000 -operation=put -requests=20000 -endpoint="https://10.96.105.5:18443" -prefix=3
+```
+
+- Starts writing objects into the default bucket `test`.
+- The bucket needs to be created from the Tenant Admin CLI or otherwise prior to running s3tester.
```

Comment:
Please drop any references to SG specifics. The previous line was OK:
"The bucket needs to be created prior to running s3tester."

Reply:
Addressed in latest commit